devel/libenkf/src/enkf_main.c

```diff
@@ -1666,6 +1684,16 @@ void enkf_main_run_exp(enkf_main_type * enkf_main ,
 }
 
 
+void enkf_main_run_create_run_path(enkf_main_type * enkf_main , bool_vector_type * iactive , int iter) {
```

Comment:
That is one `run` too much: `enkf_main_create_run_path()` should suffice; if that creates conflicts downstream those functions should be renamed.